ilastik/applets/edgeTrainingWithMulticut/edgeTrainingWithMulticutGui.py

```diff
@@ -101,3 +98,27 @@ def configure_gui_from_operator(self, *args):
     def configure_operator_from_gui(self):
         EdgeTrainingGui.configure_operator_from_gui(self)
         MulticutGuiMixin.configure_operator_from_gui(self)
+
+
+def _enable_widget_on_usable_values(widget, *slots):
```

Comment:
just for the case one would ever want to add an argument here, I would prefer there to be a `check_slots` argument that takes a list of slots....